### PR TITLE
Bradm/seo updates 2

### PR DESCRIPTION
**Pull request by bradmccarty**
More SEO updates, per our SEO/backlinks folks at Userp.

### CODE REVIEW

**Comment by mooreds** [comment]
@bradmccarty do you need someone to review?

File: astro/src/content/articles/identity-basics/magic-links.md [hunk 1]
```diff
@@ -37,6 +37,20 @@ Despite being the standard for years, passwords are a known security weak point.
 
 Implementing a secure password-based system can be a challenge. It involves creating secure password storage, implementing encryption, managing password resets, etc. In comparison, magic link systems can be simpler to implement and maintain.  They don't require any changes to existing infrastructure, which makes them a good choice for small and medium businesses. In addition, they can be implemented with fewer resources than other security solutions such as software-based two-factor authentication (2FA) systems or hardware tokens.
 
+## Are Magic Links Better Than Passwords?
+
+The comparison between the traditional login process using passwords and magic link authentication is both valid and  pertinent. Let’s explore why magic links might be a superior choice for enhancing the authentication process.
```
Comment (mark-robustelli) [inline astro/src/content/articles/identity-basics/magic-links.md:42]:
```suggestion
The comparison between the traditional login process using passwords and magic link authentication is both valid and pertinent. Let’s explore why magic links might be a superior choice for enhancing the authentication process.
```

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Mostly comments and suggestions.  1 extra space that does need to be removed.

File: astro/src/content/articles/authentication/multi-factor-authentication.md [hunk 2]
```diff
@@ -255,7 +285,7 @@ With this technique, a user splits a password into two pieces. One is stored in
 
 This method protects against the compromise of a password manager or other user side password storage mechanism. However, if the system which stores the hash of a password is compromised, or the password is discovered in transit, a horcrux won't help.
 
-### Who You Are
+### Inherence: Who You Are
```
[the remark this answers is not in the document]
Comment (bradmccarty) [reply astro/src/content/articles/authentication/multi-factor-authentication.md:288]: This came from the SEO folks. I'm familiar, but I think this is google-centric.

File: astro/src/content/articles/authentication/multi-factor-authentication.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+title: Multi-Factor Authentication (MFA): The Ultimate Guide
+description: What is multi-factor authentication and how it works?
 author: Dan Moore
 section: Authentication
 tags: mfa 2fa multi-factor-authentication two-factor-authentication user-experience customer-experience security policie sms totp factors ap-push tradeoffs open-standards biometric
 icon: /img/icons/multi-factor-authentication.svg
 ---
 
-As more of our lives move online, [multi-factor authentication (MFA)](/docs/lifecycle/authenticate-users/multi-factor-authentication) becomes increasingly important as a way of keeping our accounts secure. As a user, you know you should enable MFA on any accounts containing valuable data or which you want to keep safe. 
+As more of our lives move online, multi-factor authentication (MFA) becomes increasingly important as a way of keeping our accounts secure. As a user, you know you should enable MFA on any accounts containing valuable data or which you want to keep safe.
```
[the remark this answers is not in the document]
Comment (bradmccarty) [reply astro/src/content/articles/authentication/multi-factor-authentication.md:10]: This was a suggestion from the SEO folks. My guess is that they saw some cannibalism in search performance away from this page.

**Comment by bradmccarty** [comment]
@mark-robustelli Addressed the changes, but it's still showing a blocked merge. Not sure why.

**Review by mark-robustelli — APPROVED** [review]

**Comment by mark-robustelli** [comment]
@bradmccarty try to merge now.  I had to approve after you made the change.

**Pull request merged**